src/scss/next.scss

```diff
@@ -503,6 +503,8 @@ details summary + * {
 
 /// Form fields
 
+// See https://github.com/GoogleChrome/web.dev/issues/7638#issuecomment-1087607518
```

Comment:
I would add an explicit comment that this targets text inputs by default, and then follow by the link

Reply:
Done.